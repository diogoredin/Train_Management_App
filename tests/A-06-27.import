SERVICE|520|12.2|09:02|A1|09:03|A2|09:04|B3|09:26|A4|09:48|A5|09:57|A6|10:01|A7
SERVICE|590|12.2|07:02|A1|07:03|A2|07:04|B3|07:26|A4|07:48|A5|07:57|A6|08:01|A7
SERVICE|510|12.2|08:02|A1|08:03|A2|08:04|B3|08:26|A4|08:48|A5|08:57|A6|09:01|A7
SERVICE|790|12.2|14:02|A1|14:03|A2|14:04|B3|14:26|A4|14:48|A5|14:57|A6|15:01|A7
SERVICE|550|12.2|10:02|A1|10:03|A2|10:04|B3|10:26|A4|10:48|A5|10:57|A6|11:01|A7
SERVICE|560|12.2|11:02|A1|11:03|A2|11:04|B3|11:26|A4|11:48|A5|11:57|A6|12:01|A7
SERVICE|500|12.2|12:02|A1|12:03|A2|12:04|B3|12:26|A4|12:48|A5|12:57|A6|13:01|A7
SERVICE|690|12.2|13:02|A1|13:03|A2|13:04|B3|13:26|A4|13:48|A5|13:57|A6|14:01|A7
PASSENGER|Azura
PASSENGER|Boethiah
PASSENGER|Clavicus Vile
PASSENGER|Hermaeus Mora
PASSENGER|Hircine
PASSENGER|Malacath
PASSENGER|Mehrunes Dagon
PASSENGER|Mephala
PASSENGER|Meridia
