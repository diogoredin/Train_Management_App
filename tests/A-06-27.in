6
3
1
A1
A5
2018-12-23
05:00
0
3
1
A1
A5
2017-12-31
10:03
0
0
0
