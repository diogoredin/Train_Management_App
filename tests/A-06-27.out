Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): 
Itinerário 1 para 2018-12-23 @ 9.51
Serviço #590 @ 9.51
07:02 A1
07:03 A2
07:04 B3
07:26 A4
07:48 A5

Itinerário 2 para 2018-12-23 @ 9.51
Serviço #510 @ 9.51
08:02 A1
08:03 A2
08:04 B3
08:26 A4
08:48 A5

Itinerário 3 para 2018-12-23 @ 9.51
Serviço #520 @ 9.51
09:02 A1
09:03 A2
09:04 B3
09:26 A4
09:48 A5

Itinerário 4 para 2018-12-23 @ 9.51
Serviço #550 @ 9.51
10:02 A1
10:03 A2
10:04 B3
10:26 A4
10:48 A5

Itinerário 5 para 2018-12-23 @ 9.51
Serviço #560 @ 9.51
11:02 A1
11:03 A2
11:04 B3
11:26 A4
11:48 A5

Itinerário 6 para 2018-12-23 @ 9.51
Serviço #500 @ 9.51
12:02 A1
12:03 A2
12:04 B3
12:26 A4
12:48 A5

Itinerário 7 para 2018-12-23 @ 9.51
Serviço #690 @ 9.51
13:02 A1
13:03 A2
13:04 B3
13:26 A4
13:48 A5

Itinerário 8 para 2018-12-23 @ 9.51
Serviço #790 @ 9.51
14:02 A1
14:03 A2
14:04 B3
14:26 A4
14:48 A5
Número do itinerário: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): 
Itinerário 1 para 2017-12-31 @ 9.51
Serviço #560 @ 9.51
11:02 A1
11:03 A2
11:04 B3
11:26 A4
11:48 A5

Itinerário 2 para 2017-12-31 @ 9.51
Serviço #500 @ 9.51
12:02 A1
12:03 A2
12:04 B3
12:26 A4
12:48 A5

Itinerário 3 para 2017-12-31 @ 9.51
Serviço #690 @ 9.51
13:02 A1
13:03 A2
13:04 B3
13:26 A4
13:48 A5

Itinerário 4 para 2017-12-31 @ 9.51
Serviço #790 @ 9.51
14:02 A1
14:03 A2
14:04 B3
14:26 A4
14:48 A5
Número do itinerário: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: 